pkg/v1/cli/pluginmanager/manager.go

```diff
@@ -702,3 +708,31 @@ func getPluginDescriptorResource(pluginFilePath string) (*cliv1alpha1.PluginDesc
 	}
 	return &pd, nil
 }
+
+// verifyPlugin verifies the authenticity of the plugin source
```

Comment:
nit: saying we are verifying for authenticity is probably a stretch.
```suggestion
// verifyPlugin verifies the plugin source is allowed to be used
```
We can update the comment once the implementation changes.